src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMMA = ",";
+    private static final String NEW_LINE = System.lineSeparator();
+    private static final int OPERATION_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] dataFromFile = readFromFile(fromFileName);
+        int totalSupply = 0;
+        int totalBuy = 0;
+
+        for (String line : dataFromFile) {
+            if (!line.isEmpty()) {
+                String[] parts = line.split(COMMA);
+                String operationType = parts[OPERATION_INDEX];
+                int amount = Integer.parseInt(parts[AMOUNT_INDEX]);
+
+                if (SUPPLY.equals(operationType)) {
+                    totalSupply += amount;
+                } else if (BUY.equals(operationType)) {
+                    totalBuy += amount;
+                }
+            }
+        }
+
```

Comment:
encapsulate into a separate method too ,so that your public method is clean and short